src/models/msg.js

```diff
@@ -63,4 +63,9 @@ Msg.Type = {
 	BANLIST: "ban_list",
 };
 
+Msg.NonLoggableTypes = {};
+Msg.NonLoggableTypes[Msg.Type.BANLIST] = true;
+Msg.NonLoggableTypes[Msg.Type.MOTD] = true;
+Msg.NonLoggableTypes[Msg.Type.WHOIS] = true;
```

Comment:
```
Msg.NonLoggableTypes = { [Msg.Type.BANLIST]: true, ...}
```
maybe?

Reply:
Doesn't work on node 4 afaik.

Reply:
```sh
$ nvm use 4
Now using node v4.8.4 (npm v2.15.11)
$ node
> a = 42
42
> { [a]: 1}
{ '42': 1 }
```

:man_shrugging:

Reply:
Why not add a method to the `Msg` class instead that would do:

```js
function isLoggable() {
	return this.type !== Msg.Type.MOTD &&
		this.type !== Msg.Type.BANLIST &&
		this.type !== Msg.Type.WHOIS;
}
```

And in `writeUserLog`:

```js
if (!msg.isLoggable()) {
	return false;
}
```

It seems more idiomatic to me, no?
(I went for `isLoggable` and not `isNotLoggable` because it's fairly common practice to avoid negated boolean functions)

Bonus, you can get rid of `const Msg = require("./msg");` 😁